cgmanifest.json

```diff
@@ -4426,7 +4426,7 @@
         "other": {
           "name": "ncurses",
           "version": "6.3",
-          "downloadUrl": "https://invisible-mirror.net/archives/ncurses/ncurses-6.3.tar.gz"
+          "downloadUrl": "https://invisible-mirror.net/archives/ncurses/current/ncurses-6.3-20220612.tar.gz"
```

Comment:
The `current` in this URL makes me think it'll break when 6.4 is released- is there a more permanent URL we can use here?

Reply:
alternately, we could continue to use the original 6.3 source and check in the patch. (http://invisible-mirror.net/archives/ncurses/6.3/ncurses-6.3-20220612.patch.gz)

Reply:
FWIW, I wasn't able to find a more permanent URL for the patched sources

Reply:
I was reluctant to take just the patch....preferred to take the accumulated.  
And...yes, I think a better link is more desirable, but letting it go as is.

Reply:
Sounds good- I agree that using patched sources is cleaner despite the issues.